src/appshell/view/appmenumodel.cpp

```diff
@@ -72,6 +74,15 @@ void AppMenuModel::load()
     appMenuModelHook()->onAppMenuInited();
 }
 
+bool AppMenuModel::nativeMenuBarAvailable()
+{
+    // When any QMenuBar is created, it sets the nativeMenuBar property to true
+    // if it can be used as the native menu bar.
+    // This dummy menu bar is created just to check native menu bar availability.
+    QMenuBar dummyMenuBar;
+    return dummyMenuBar.isNativeMenuBar();
```

Comment:
I'm not a big fan of creating such a dummy object; in the case of QMenuBar that doesn't look like a particularly trivial operation, since the underlying menu bar is immediately created, only to be destroyed a few nanoseconds later, when the method returns. 

It would be preferable to use just the code that detects whether the native menu bar will be created. I took a quick look at the Qt source code, and it seems to boil down to this function: https://codebrowser.dev/qt5/qtbase/src/platformsupport/themes/genericunix/qgenericunixthemes.cpp.html#118

Reply:
Ok, thanks, I should have looked into the source code...
By the way, is it okay to add this function to AppMenuModel? I think it's the most appropriate class for it.

Reply:
Yes, that seems fine to me!

Reply:
To use D-Bus, MuseScore has to link with the Qt D-Bus module. I'm not sure if it's enough to just add it to `build/cmake/FindQt5.cmake`, though.